science/luaradio/Portfile

```diff
@@ -3,11 +3,11 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        vsergeev luaradio 0.6.1 v
+github.setup        vsergeev luaradio 0.7.0 v
 
-checksums           rmd160  7bc5090eb17111719a819460ecefffdea79fe20e \
-                    sha256  8f5e539d84d70717a7bf18cce2dc2416bcfbed3a561d869ff66af47c049705a6 \
-                    size    9187436
+checksums           rmd160  ec6f53ba74d0ce182b297e020c5ca1251c2fb534 \
+                    sha256  081eab02877fa6060c9c8a3cd46028358a8b61c78c5d54dfed0fb0c333296fb6 \
+                    size    9194289
 revision            1
```

Comment:
Please always reset the revision to 0 on a version update.

I would also suggest moving this line to be just below the github setup line, as I think having it closer to were the version is defined makes it easier to remember to reset it.